pkg/harbor/registry.go

```diff
@@ -109,7 +109,7 @@ func (s *registry) do(req *http.Request) (*http.Response, error) {
 	resp.Body = buf
 
 	if resp.StatusCode < 200 || resp.StatusCode >= 300 {
-		s.logger.V(-1).Info("HTTP response status code is not OK",
+		s.logger.V(1).Info("HTTP response status code is not OK",
```

Comment:
it is modified because we should not bother the users with status code in non-verbose mode